src/Common/src/Internal/Runtime/CompilerHelpers/StartupCodeHelpers.cs

```diff
@@ -134,6 +96,8 @@ private static unsafe void InitializeGlobalTablesForModule(IntPtr moduleManager)
             }
         }
 
+        static unsafe partial void InitializeStringTable(IntPtr stringTableStart, int length);
```

Comment:
I took out string literal support because the UTF8 decoder brings more things than I would want it to. This will go away when we get support for frozen strings anyway.